cmd/kube-scheduler/app/server.go

```diff
@@ -20,6 +20,8 @@ package app
 import (
 	"context"
 	"fmt"
+	"k8s.io/component-base/metrics/features"
+	"k8s.io/component-base/metrics/prometheus/slis"
```

Comment:
The imports are generally groups into three categories: std-lib, third-party, followed by k8s dependencies.